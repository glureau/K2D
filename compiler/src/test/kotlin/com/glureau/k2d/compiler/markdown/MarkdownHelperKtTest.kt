package com.glureau.k2d.compiler.markdown

import org.approvaltests.Approvals
import org.junit.Test

internal class MarkdownHelperKtTest {

    @Test
    fun fo() {
        Approvals.verify(mdTable(
            headers = listOf("name", "type", "comments"),
            listOf("name", "String", ""),
            listOf("productUuid", "Int", "no comments"),
            listOf("ez", "List<String>", "a"),
        ))
    }
}