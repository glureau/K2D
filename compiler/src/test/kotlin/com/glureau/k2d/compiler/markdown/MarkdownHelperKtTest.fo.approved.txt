| name        | type         | comments    |
|-------------|--------------|-------------|
| name        | String       |             |
| productUuid | Int          | no comments |
| ez          | List<String> | a           |

